starter/angular.json

```diff
@@ -84,9 +109,11 @@
           }
         },
         "serve": {
-          "builder": "@angular-devkit/build-angular:dev-server",
+          "builder": "@angular-builders/dev-server:generic",
           "options": {
-            "browserTarget": "starter:build"
+            "browserTarget": "starter:build",
+            "port": 3000,
+            "open": true
           },
           "configurations": {
             "hmr": {
```

Comment:
To make hmr working, please set it like this (according to the official [doc](https://github.com/angular/angular-cli/wiki/stories-configure-hmr)):
```json
"hmr": {
  "hmr": true,
  "browserTarget": "showcase:build:hmr"
}
```